### PR TITLE
React version of ff_container-form-line - ff_module-form-label - ff_module-form-input

### PR DESCRIPTION
**Pull request by AdamLynamFirefly**

### CODE REVIEW

**Comment by dkhuntrods** [comment]
Looks good, if you could add some tests we can merge

File: tests/ff_module-form-input.js [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+        var component = TestUtils.renderIntoDocument(element);
+        var root = TestUtils.findRenderedDOMComponentWithClass(component, 'ff_module-form-input');
+        expect(root.tagName).to.equal("SELECT");
+        expect(root.getElementsByTagName('option')[0].innerHTML).to.equal(optionText);
+        expect(root.getElementsByTagName('option')[0].getAttribute("value")).to.equal(optionValue);
+	});
+    
+    // [onClick, onChange]
+    // check onClick and onChange event handlers
+    var onClickRan = false,
```
Comment (dkhuntrods) [inline tests/ff_module-form-input.js]: Might be easier to use Sinon here?

Comment (AdamLynamFirefly) [inline tests/ff_module-form-input.js]: I am not sure what that means. I wasn't even able to Google it. :/

This check felt a bit clunky, but I wasn't sure how else to do it.

Comment (dkhuntrods) [inline tests/ff_module-form-input.js]: Ah, there's a module in the tests already that allows you to do this - anywhere you see sinon.spy() that's what's going on. You can see docs here: http://sinonjs.org/

File: blocks/core/ff_container/ff_container-form-line/ff_container-form-line.js [hunk 2]
```diff
@@ -0,0 +1,14 @@
+'use strict';
+
+var React = require('react');
+
+module.exports = React.createClass({
+	displayName: 'FormLabel',
```
Comment (dkhuntrods) [inline blocks/core/ff_container/ff_container-form-line/ff_container-form-line.js]: Is that the correct name?

**Pull request merged**